docs/user-guide/widgets.md

```diff
@@ -67,27 +93,30 @@ Just below the chart's preview, the following configurations are available:
+The user can customize the style with the following options choosing between `Simple style` or `Classification style`
 
-<img src="../img/widgets/custom_colors_settings.jpg" class="ms-docimage"/>
+With the **Simple style**, the editor is allowed to customize the `Fill color`, the `Outline color` and the `Outline width`
 
-Inside this modal, the user is allowed to:
+<img src="../img/widgets/trace_simple_style.jpg" class="ms-docimage"  style="max-width:450px;"/>
 
-* Change the default **Color** of bars or slices (depending on the chart type) through the *Color Picker*. This color will be applied for all values for which a *Class Color* has not been configured.
+###### Classification style
 
-<video class="ms-docimage" controls><source src="../img/widgets/custom_colors.mp4"/></video>
+For *Bar Charts* and *Pie Charts*, [MapStore](https://mapstore.geosolutionsgroup.com/mapstore/#/) provides the possibility to customize the colors of the charts bars and slices. It is possible to classify the style based on the attributes of the layer. In this case it is possible to select the `Attribute` to use for the classification along with the classification `Method` (*Quantile*, *Equal interval*, *Natural breaks* and *Unique interval*) and select whether to `Sort` the classification by *Attribute X* or *Attribute Y*. Choose a `Color ramp`, the number of classification `Intervals` and the order (with `Reverse order`) of the classification intervals colors. Finally customize the `Outline color` and the `Outline width`.
+
+<img src="../img/widgets/trace_classification_style.jpg" class="ms-docimage"  style="max-width:450px;"/>
 
-* Select an *Attribute* in the dropdown list as a **Classification attribute**.
+From the **Color Ramp** option, the user can select the *Custom colors* option and open the **Custom Colors Settings** modal through the <img src="../img/button/edit_button.jpg" class="ms-docbutton"/> button.
+Inside this modal, the user is allowed to:
 
-<video class="ms-docimage" controls><source src="../img/widgets/classification-attribute.mp4"/></video>
+* Change the default **Color** of bars or slices (depending on the chart type) through the *Color Picker*. This color will be applied for all values for which a *Class Color* has not been configured.
 
-Once the attribute is chosen, new options appear in the *Custom Color Settings* panel that allow the user to:
+<video class="ms-docimage" controls><source src="../img/widgets/custom_colors.mp4"/></video>
 
 * Enter a **Default Class Label** to be used in the legend for all values that will not be specifically classified in the following list.
```

Comment:
Same of [this](https://github.com/geosolutions-it/MapStore2/pull/9677/files#diff-fa9572d42754707058739c40bac414046b2e986d1dbf35a065d141e26fe71bf7R112) here.

Reply:
@ElenaGallo what about this one?

Reply:
Added a new image with commit: [review_9669](https://github.com/geosolutions-it/MapStore2/pull/9677/commits/a4148f53c053eea7453c8fe5387910d2ccc2d122)

Reply:
@ElenaGallo correct if I'm wrong but this change doesn't appear in the doc build:
https://mapstore--9677.org.readthedocs.build/en/9677/user-guide/widgets/

did I miss something?